grpc/pb-marshalling.go

```diff
@@ -193,8 +193,10 @@ func ValidationResultToPB(records []core.ValidationRecord, prob *probs.ProblemDe
 		return nil, err
 	}
 	return &vapb.ValidationResult{
-		Records:  recordAry,
-		Problems: marshalledProbs,
+		Records:     recordAry,
+		Problems:    marshalledProbs,
+		Perspective: perspective,
+		Rir:         rir,
```

Comment:
I have not handled these fields being present in `pbToValidationResult()` below. Ultimately I believe this is fine as we don't depend on them, but it could be a little surprsing.